app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -23,4 +23,10 @@ def self.event_monitor_class
   def supported_auth_attributes
     %w(userid password auth_key)
   end
+
+  def create_project(project)
```

Comment:
@bzwei `project` here is a hash that represents the project to be created in openshift and then collected and saved in VMDB table `container_projects`.